Classes/Public/Manager/Objects/OCTMessageCall.h

```diff
@@ -6,19 +6,19 @@
 //  Copyright (c) 2015 dvor. All rights reserved.
 //
 
-#import "OCTMessageAbstract.h"
+#import "OCTObject.h"
 #import "OCTManagerConstants.h"
 
-@interface OCTMessageCall : OCTMessageAbstract
+@interface OCTMessageCall : OCTObject
 
 /**
  * The length of the call in seconds.
  **/
-@property (assign, nonatomic, readonly) NSTimeInterval callDuration;
+@property  NSTimeInterval callDuration;
```

Comment:
On the other hand call is "temporary" object that lives only while active while message is "eternal" object representing use history. I was wrong about adding one to another.

Also I don't really like that the message is created once call is created. In my opinion there is no need in a message while we have a call. Only when "active" call is ended it should be transformed into "history" message. So what do you think about following:
- leave OCTMessageCall as it is now
- don't create message on incoming/outgoing call
- when call gets ended
  - create message from it
  - remove the call
- maybe change `OCTCallStatusIncoming` -> `OCTCallStatusRinging`, `OCTCallStatusInSession` -> `OCTCallStatusActive`?
- there is no need in `OCTCallStatusInactive` status for call. It can be only in *dialing*, *ringing* or *active* states.
- there is no need in `OCTMessageCallEventDial` and `OCTMessageCallEventStarted` for message. It can only be *answered* or *unanswered*
- maybe chage `OCTMessageCallEventEnd` -> `OCTMessageCallAnswered`, `OCTMessageCallEventMissed` -> `OCTMessageCallUnanswered`? This way it would have clear naming for both incoming and outgoing calls

Also there may be a crash during the call or app can be simply killed by user. To handle that case we can loop through all calls on startup, transform them into messages and then delete.